src/main/java/org/embulk/base/restclient/jackson/JacksonServiceResponseMapper.java

```diff
@@ -141,10 +151,10 @@ else if (type.equals(Types.STRING)) {
         }
         else if (type.equals(Types.TIMESTAMP)) {
             TimestampParser timestampParser = new TimestampParser(
-                Exec.newConfigSource().loadConfig(TimestampParser.Task.class),
+                Exec.newConfigSource().loadConfig(InternalTimestampParserTask.class),
```

Comment:
This `loadConfig` fails because `org.embulk.spi.time.TimestampParser.Task` does not extend `org.embulk.config.Task`.